locale/en/get-involved/node-meetups.md

```diff
@@ -26,8 +26,281 @@ REQUIREMENTS
 - If you do not currently have a CoC, update the meetup with CoC before submitting.
 - Submit your PR in alphabetical order.
 
-## Meetups
```

Comment:
i wonder why this line was deleted?

Reply:
Thought maybe it was obvious but no worries, will add again!